translations/trading.tex/pl.po

```diff
@@ -37,31 +37,36 @@ msgstr ""
 
 #.  !TeX spellcheck = en_US
 #. type: Plain text
-#: sections/other_rules.tex:4
-msgid "\\addsection{Other Rules}{\\skills/armorer.png}"
-msgstr "\\addsection{Inne Zasady}{\\skills/armorer.png}"
+#: sections/trading.tex:4
+#, fuzzy
+#| msgid "\\addsection{Town}{\\skills/artillery.png}"
+msgid "\\addsection{Trading}{\\skills/armorer.png}"
+msgstr "\\addsection{Miasto}{\\skills/artillery.png}"
```

Comment:
@qwrtln @evilmint I don't recall changing this file, I just moved them with git mv, and somehow po4a made this change. Does it look correct to you?